### PR TITLE
[ros2]: rviz_marker_tools: Fix dependencies & modernize CMake

### PR DESCRIPTION
**Pull request by JafarAbdi**

### CODE REVIEW

**Comment by henningkayser** [comment]
@JafarAbdi any idea about the asan failure? seems to be caused by rcl

File: rviz_marker_tools/package.xml [hunk 1]
```diff
@@ -15,6 +15,7 @@
 	<depend>geometry_msgs</depend>
 	<depend>rclcpp</depend>
 	<depend>rviz2</depend>
+	<depend>urdfdom_headers</depend>
```
Comment (AndyZe) [inline rviz_marker_tools/package.xml:18]: Indentation and alphabetization is messed up here

**Review by AndyZe — APPROVED** [review]
I've tested this and it fixes my ROS2 Rolling build. Thanks a lot!

**Comment by AndyZe** [comment]
Let's get this merged since you're on vacation and I'll file a follow-up PR to alphabetize and fix the indentation

**Pull request merged**